torch/testing/_internal/common_utils.py

```diff
@@ -970,6 +970,25 @@ def assertAlmostEqual(self, x, y, places=None, msg='', delta=None, allow_inf=Non
             prec = 10**(-places)
         self.assertEqual(x, y, msg, atol=prec, allow_inf=allow_inf)
 
+    def assertQuantizedClose(self, qx, qy, tol_off_by_y_scale=1e-2, message=''):
```

Comment:
Also, can we call the argument fraction_of_one_off_error instead of tol_off_by_y_scale

Reply:
the current name is consistent with other similar arguments in this file, but i'm flexible, especially if this is moved elsewhere as Jerry is suggesting